contents/articles/2016-11_van-gogh-museum-project/index.md

```diff
@@ -0,0 +1,23 @@
+--- 
+title: The Van Gogh Museum Eye-tracking Project
+date: Wed Nov 02 2016 10:07:55 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "Researchers at the VU Amsterdam in collaboration with The Van Gogh Museum attemps to answer how we perceive art..."
+featured_img: "../../../../media/images/blog/van-gogh-museum.jpg"
+featured_img_thumb: "../../../../media/images/blog/thumb/van-gogh-museum.jpg"
+---
+
+<img src="../../../../media/images/blog/van-gogh-museum.jpg" class='Feature-image u-padBottom--1' alt="The Van Gogh Museum Eye-tracking Project">
+Image Source: [The Van Gogh Museum Eye-tracking Project](http://www.vupsy.nl/van-gogh-museum-eye-tracking-project/)
+
+<br>
+A group of researchers from the [Department of Experimental and Applied psychology](http://www.vupsy.nl/) at the [VU Amsterdam](http://www.vu.nl/en/) in collaboration with the [Van Gogh Museum](https://www.vangoghmuseum.nl/en) attempted to answer how we perceive and appreciate art in a 'real life' environment. Previous studies are often set in a laboratory environment in which people are stationary with bulky equipments. Surely this is not people normally view paintings. So how do we conduct the study in a way that is more of a 'natural setting'?
```

Comment:
First sentence modification
We are really excited to see Pupil used in research from...